environments/mamba_full_package_list_1.13.0_20240724.yml

```diff
@@ -0,0 +1,222 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
+_libgcc_mutex=0.1=conda_forge
+_openmp_mutex=4.5=2_gnu
+_sysroot_linux-64_curr_repodata_hack=3=h69a702a_16
+accessible-pygments=0.0.5=pypi_0
+affine=2.4.0=pypi_0
+akima86=0.1.1=pypi_0
+alabaster=0.7.16=pypi_0
+annotated-types=0.7.0=pypi_0
```

Comment:
Do we want to add these to the YAML release note?  Maybe it is good to track per-PR when we create these environment dumps?  Along with a copy/paste of the full-test output, showing that these dependency versions do in fact relate to a complete 0 return for everything?

Reply:
I kind of like tracking this, but that environemnts directory is going to get rather cluttered!!!

Reply:
Might not be the worst idea. I'll leave that up to you and @jsolbrig.

Reply:
I say we discuss this later and just get this merged as is.